joss/paper.md

```diff
@@ -86,17 +86,19 @@ for a large scale experiment. The combinatorial explosion is
 unavoidable; `fitgrid` contains it.
 
 The rERP approach is attracting growing attention in the field but to
-date the open-source EEG and MEG data analysis landscape has been
+date the open-source EEG and magnetoencephalography (MEG) data analysis
+landscape has been
 dominated by toolboxes written for MATLAB such as EEGLAB
 [@DelMak2004], FieldTrip [@OosEtAl2011], and Brainstorm
-[@TadEtAl2011], and this holds for rERP modeling, e.g., @EhiDim2019.
+[@TadEtAl2011], and this holds for rERP modeling (e.g., @EhiDim2019)s.
 Like open-source scientific computing generally, Python and R have
-been gaining traction for EEG and MEG analysis, as in MNE Python
-@GramfortEtAl2013 and for regression ERPs in R,
+been gaining traction for EEG and MEG analysis, as in MNE-Python
+@GramfortEtAl2013, and for regression ERPs in R
 @TreNew2015. Nevertheless, widely accessible implementations for rERP
-modeling in the Python remain limited.  Development of N. J. Smith's
+modeling in Python remain limited. Development of N. J. Smith's
 promising rERPy Python package for ERP and rERP analysis appears to
-have halted in Python 2.X. MNE Python implements a `linear_regression`
+have halted in Python 2, which reached its end of life in January 2020.
```

Comment:
would be nice to get a link to that project here (even if apparently outdated)

Reply:
Done